Hierarchical clustering of PCAWG samples

dmaziec1 opened the pull request:
Clustering script added. 
I added the output results, which we used for uploading to allow users to review generated heat maps and other files. Please let me know if it's ok to store it here in the repository or I should move it somewhere else and provide a link.

sehilyi commented on scripts/clustering/README.md:
````diff
@@ -0,0 +1,50 @@
+The help command of the tool: 
+
+```
+python3 clustering.py  --help 
+Usage: clustering.py [OPTIONS]
````
I think you can also provide here the commands you used to generate output files. (Maybe I missed it!)

sehilyi reviewed (approved):
These are great -- Thanks a lot for adding this!

dmaziec1 commented:
Hi @sehilyi, 
I addressed your comments:
- updated the README file 
- added `run_clustering.sh` -- downloads the excel file with the signatures and runs the script
- adjusted in `clustering.py` loading the excel file as a data frame

sehilyi commented:
@dmaziec1 Thanks for the update! Feel free to merge!
